### DISCUSSION TITLE
Is possible to found pdo_mysql and other drivers

### DISCUSSION THREAD

**Discussion by filipagh** [composer-unused/composer-unused#364] [Ideas]
`ext-pdo` is marked as used  but `ext-pdo_mysql` no 

like `PDO::MYSQL_ATTR_USE_BUFFERED_QUERY` which is valid only for `ext-pdo_mysql`

i think this should be detect somehow

**Comment by icanhazstring** [comment]
Hm. This should be detected. Maybe the matching of those constants is not working as the name of the extensions is pretty similar. Will investigate.
